src/monet.js

```diff
@@ -177,6 +177,22 @@
         return listFindC(l, fn).run()
     }
 
+    function listContainsC(l, val) {
+        if (l.isNil) {
+            return Return(false)
+        }
+        var h = l.head()
+        return h === val ?
```

Comment:
I think that we should add some helper function so we can use `.equals()` powers of our structures. It'll also work with other 3rd party stuff that implement [Setoid](https://github.com/fantasyland/fantasy-land#setoid).

See [Immutable.JS `is()`](https://facebook.github.io/immutable-js/docs/#/is)